### PR TITLE
googleSheets update-rows component

### PR DESCRIPTION
**Pull request by dreammonkey**
It seems this component was not yet implemented.

The component accepts a range argument, the values in range will be overwritten with the given rows.

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/3413"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

File: components/google_sheets/actions/update-rows/update-rows.mjs [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import googleSheets from "../../google_sheets.app.mjs";
+
+export default {
+  key: "google_sheets-update-rows",
+  name: "Update Rows",
+  description: "Update multiple rows in a spreadsheet defined by a range",
+  version: "0.0.4",
```
Comment (feyzullah) [inline components/google_sheets/actions/update-rows/update-rows.mjs:7]:
```suggestion
  version: "0.0.1",
```
Because this is the first version

**Review by feyzullah — CHANGES_REQUESTED** [review]
Hi @dreammonkey, thank you very much for your contribution. I added just a few comments, mostly about linter errors.
I guess your repo is behind `PipedreamHQ/pipedream` and if you would like to sync and merge after QA checks, the following might be useful;
```git remote add upstream https://github.com/PipedreamHQ/pipedream.git
git fetch upstream
git merge upstream/master
# from repo root
npx pnpm install -r
git add pnpm-lock.yaml
git commit
git push
```

**Comment by dannyroosevelt** [comment]
@feyzullah can you just implement these changes, since they're pretty tactical / specific to our style guidelines?

**Comment by feyzullah** [comment]
> @feyzullah can you just implement these changes, since they're pretty tactical / specific to our style guidelines?

Yes sure

**Comment by feyzullah** [comment]
Hi @dreammonkey, I created [another PR](https://github.com/PipedreamHQ/pipedream/pull/3426) for this one which is in QA process now. If you would like to do please subscribe to it.

**Comment by feyzullah** [comment]
Hi @dreammonkey, it should have been published. Can you please check?

**Pull request closed without merging**